src/App.js

```diff
@@ -1,6 +1,41 @@
 import React from 'react';
+import { Routes, Route, Link } from 'react-router-dom';
+// import Calculator from './components/calculator';
```

Comment:
- [ ] Kindly please remove all lines of old commented pieces of code, so the project gets cleaner 💪

Reply:
done